tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateEnums.java

```diff
@@ -79,6 +79,9 @@ public class GenerateEnums {
 
     private Set<String> languages = new TreeSet<>();
 
+    private final Set<String> ignoreMissingAlpha3 =
+            new TreeSet<>(Arrays.asList("EA", "EZ", "IC", "UN"));
```

Comment:
More changes like this would belong in https://unicode-org.atlassian.net/browse/CLDR-17637